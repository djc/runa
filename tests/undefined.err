undefined.rns [2.11]: undefined name 'x'
    print(x)
          ^
