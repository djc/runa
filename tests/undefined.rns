def main():
	print(x)
	return 0
