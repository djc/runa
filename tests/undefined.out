tests/undefined.lng [2.8]: undefined name 'x'
    print(x)
          ^
